src/components/comment/Comment.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../styles/variables';
+
+.bdl-Comment {
+    font-size: .875rem;
```

Comment:
We don't typically use `rem` for our font sizes. Can we use the `bdl-fontSize` variables?

Reply:
Sure, I haven't noticed `bdl-fontSize` variables. There wasn't one for 12 px, so I've created it. I'm not sure if there is any particular naming convention though (I've named it `$bdl-fontSize-medium`).